lib/ansible/modules/packaging/os/yum.py

```diff
@@ -631,7 +643,18 @@ def list_stuff(module, repoquerybin, conf_file, stuff, installroot='/'):
     elif stuff == 'repos':
         return [ dict(repoid=name, state='enabled') for name in sorted(repolist(module, repoq)) if name.strip() ]
     else:
-        return [ pkg_to_dict(p) for p in sorted(is_installed(module, repoq, stuff, conf_file, qf=is_installed_qf, installroot=installroot) + is_available(module, repoq, stuff, conf_file, qf=qf, installroot=installroot)) if p.strip() ]
+        return [ pkg_to_dict(p) for p in sorted(is_installed(module,
+                                                             repoq,
+                                                             stuff,
+                                                             conf_file,
+                                                             qf=is_installed_qf,
+                                                             installroot=installroot) +
+                                                is_available(module,
+                                                             repoq,
+                                                             stuff,
+                                                             conf_file,
+                                                             qf=qf,
+                                                             installroot=installroot)) if p.strip() ]
```

Comment:
This is horrible formatting ;-)  If you want to break this up, make logical divisions more like this:

``` python
return [ pkg_to_dict(p) for p in sorted(
                  is_installed(module, repoq, stuff, conf_file, qf=is_installed_qf, installroot=installroot) +
                  is_available(module, repoq, stuff, conf_file, qf=qf, installroot=installroot))
            if p.strip() ]
```